src/scripts/selectors/UserSelectors.js

```diff
@@ -19,3 +19,6 @@ export const getIsLoggedIn: (state: RootState) => boolean = createSelector(
   (user: User, isLoggingIn: boolean): boolean =>
     !!(user.get('email') && !isLoggingIn)
 )
+
+export const getAddress = (state: RootState): string =>
+  createSelector([getWallet], (wallet: Wallet): string => wallet.get('address'))
```

Comment:
Here is the real payoff for this PR 😄 .

Now we can use this selector to get the wallet address and make a call to `paratii-lib` to fetch, for example, PTI balance

Reply:
Ok, butbut  have `paratii.config.accounts.address` already, which must be set for paratii-lib to function, and which will **whatever is in the redux state**, be the address with wich transactions are signed, etc.

Reply:
![image](https://user-images.githubusercontent.com/7697924/36030975-2f7e61f0-0d77-11e8-9183-5f02a7292822.png)

Then why do we need to pass in an address to this call?

Reply:
because we may be interested in the balances of other accounts as well as our own. (In theory, I have no use case yet, but you could imagine stuff like "I'm only sending you my video if I know you can pay for it).